### PR TITLE
Add LinkerD

### PR DESCRIPTION
**Pull request by Sierra1011**
Hi again,
Noticed that LinkerD wasn't on the list - hope I've got this right.
Thanks!

### CODE REVIEW

**Comment by Sierra1011** [comment]
Probably worth noting that I haven't included the {{ .ExeExtension }} suffix as a result of #56 , but the package is available for Windows.

File: DISTRIBUTIONS.md [hunk 1]
```diff
@@ -149,6 +149,7 @@
 - [kubetail](https://github.com/jbvmio/kubetail/): Repository for kubetail - The pod log tailer
 - [kubeval](https://github.com/instrumenta/kubeval/): Validate your Kubernetes configuration files, supports multiple Kubernetes versions
 - [kustomize](https://github.com/kubernetes-sigs/): Customization of kubernetes YAML configurations
+- [linkerd](https://github.com//linkerd/linkerd2/):Linkerd is an ultralight, security-first service mesh for Kubernetes. Linkerd adds critical security, observability, and reliability features to your Kubernetes stack with no code change required.
```
Comment (Cleming) [inline DISTRIBUTIONS.md:152]: Hi @Sierra1011 ,
There is a space missing 
```
:Linkerd
 ^
```

Comment (Sierra1011) [reply DISTRIBUTIONS.md:152]: :cry:

Comment (Cleming) [reply DISTRIBUTIONS.md]: thank you @Sierra1011 :)

**Comment by leucos** [comment]
thanks @Sierra1011 & @Cleming

**Comment by leucos** [comment]
and sorry for the wait

**Pull request merged**